osclib/accept_command.py

```diff
@@ -140,8 +194,32 @@ def accept_other_new(self):
             self.create_new_links(self.api.project, req['packages'][0], oldspecs)
             changed = True
 
+        if self.api.delreq_monitor:
+            rqlist = self.find_virtual_accepted_requests(self.api.project)
+            for req in rqlist:
+                if self.virtual_accept_request_has_no_binary(self.api.project, req['packages'][0]):
+                    print 'Accepting request %d: %s' % (req['id'], ','.join(req['packages']))
+                    # Accepting delreq-monitor review
+                    self.api.do_change_review_state(req['id'], 'accepted',
+                                                    by_group=self.api.delreq_monitor,
+                                                    message='Virtual accepted to delete {}'.format(req['packages'][0]))
+                    # Delete sub-package links before accepting the request
+                    self.delete_obsoleted_links(self.api.project, req['packages'][0], str(req['id']))
+                    change_request_state(self.api.apiurl, str(req['id']), 'accepted', message='Accept to %s' % self.api.project)
+                    changed = True
+
         return changed
 
+    def delete_obsoleted_links(self, project, pkgname, request_id):
+        filelist = self.api.get_filelist_for_package(pkgname=pkgname, project=project, expand='1', extension='spec')
```

Comment:
Another case for that method and the `spec[:-5]`. Definitely should be extracted as a stand-alone method.